css/styles.css

```diff
@@ -0,0 +1,267 @@
+  margin: 0 auto;
+  margin-top: 3rem;
+  background-image: linear-gradient(to left bottom, #112031, #253b4e, #3b586c, #51778c, #6998ab);
+  /* background-image: linear-gradient(to right top, #112031, #172e3e, #1f3d4b, #284c58, #345b63); */
+  border-radius: 15px;
+  height: 800px;
+  box-shadow: rgba(0, 0, 0, 0.05) 0px 1px 2px 0px;
+}
+
+.dot-img {
+  max-width: 10rem;
+}
+
+#header .header-content {
+  max-width: 550px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+h1 {
+  font-size: 1rem;
+  color: #fbfcfc;
+  margin-left: -3rem;
+}
+
+.dot {
+  margin-left: -1rem;
+  margin-right: -1.5rem;
+
+}
+
+.content-grid {
+  display: grid;
+  grid-template-columns: 25% 71%;
+  margin-top: 7rem;
+}
+
+.content-grid .grid1-employee img {
+  display: block;
+  margin: 0 auto;
+  margin-bottom: 3.5rem;
+}
+
+.content-grid .grid1-employee #form input {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 342px;
+  background-color: #2c3841;
+  border: none;
+  border-radius: 10px;
+  height: 2.5rem;
+  text-align: center;
+  color: #90969b;
+  font-weight: 400;
+  font-size: 1.2rem;
+  box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px;
+  margin-bottom: 1.8rem;
+}
+
+.content-grid .grid1-employee #form .buttons {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  
+}
+
+.content-grid .grid1-employee #form .buttons button {
+  width: 150px;
+  border: none;
+  /* background-color: #89a29f; */
+  background-color: #2c3841;
+  padding: 1rem;
+  border-radius: 12px;
+  color: #dfe6e9;
+  font-weight: 700;
+  cursor: pointer;
+  box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px;
+}
+
+.content-grid .grid1-employee #form .buttons button i {
+  margin-left: .4rem;
+}
+
+.content-grid .grid2-clock .clock-info {
+  color: #dfe6e9;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 6rem;
+}
+
+.content-grid .grid2-clock .clock-info .month {
+  font-size: 1.5rem;
+  margin-bottom: 1rem;
+}
+
+.content-grid .grid2-clock .clock-info .clock {
+  font-size: 6.8rem;
+  margin-bottom: 1rem;
+}
+
+.content-grid .grid2-clock .clock-info .time {
+  font-size: 1.5rem;
+}
+
+.content {
+  padding-left: 3.5rem;
+}
+
+/* -------- ADMIN STYLES ---------*/
+
+.content-nogrid .grid1-employee {
+  margin-top: 7.5rem;
+}
+
+.content-nogrid .grid1-employee #form .button-admin {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  
+}
+
+.content-nogrid .grid1-employee #form .button-admin button {
+  margin: 0 auto;
+  width: 150px;
+  border: none;
+  background-color: #89a29f;
+  padding: 1rem;
+  border-radius: 12px;
+  color: #ecf0f1;
+  font-weight: 700;
+  cursor: pointer;
+  box-shadow: rgba(0, 0, 0, 0.12) 0px 1px 3px, rgba(0, 0, 0, 0.24) 0px 1px 2px;
+}
+
+/* .content-nogrid {
+  display: grid;
+  grid-template-columns: 25% 71%;
+  margin-top: 7rem;
+} */
```

Comment:
If this code are not necessary, can you remove it, what do you think?

Reply:
Totally agree with @JuanequeX